src/hypermedia-constants.js

```diff
@@ -758,7 +758,8 @@ export const Actions = {
 			mergeCourseOfferings: 'merge-course-offerings',
 			searchCourseOfferings: 'search-course-offerings',
 			select: 'select',
-			selectAsTarget: 'select-as-target'
+			selectAsTarget: 'select-as-target',
+			unmergeCourseOfferings: 'unmerge-course-offerings'
```

Comment:
I would assume this is a good name for the action, unless we already had something else.

Reply:
Needs to match what you have in the unmerge PR. I left a [comment](https://github.com/Brightspace/lms/pull/31017#issuecomment-1380514063) that we forgot to add this to that PR